fixtures/node-app-pages/package.json

```diff
@@ -1,36 +1,37 @@
 {
```

Comment:
This was unformatted, I let the formatter take the wheel. The only change is line 10